python/cudf/cudf/core/dataframe.py

```diff
@@ -90,10 +91,8 @@
 
 class _DataFrameIndexer(_FrameIndexer):
     def __getitem__(self, arg):
-        from cudf import MultiIndex
-
-        if isinstance(self._frame.index, MultiIndex) or isinstance(
-            self._frame.columns, MultiIndex
+        if isinstance(self._frame.index, cudf.MultiIndex) or isinstance(
+            self._frame.columns, cudf.MultiIndex
```

Comment:
Since this file imports `from cudf.core.multiindex import MultiIndex`, this doesn't need the `cudf` module prefix. If we want to keep the `cudf` module prefix, then we can remove the `from cudf.core.multiindex import MultiIndex` and rely on the `import cudf`.
```suggestion
        if isinstance(self._frame.index, MultiIndex) or isinstance(
            self._frame.columns, MultiIndex
```

This comment applies to a few other changes to `cudf.DataFrame` below, as well. There appears to be inconsistency within this PR about whether to add or remove `cudf` module prefixes.

Overall I'm in favor of removing the module prefix and using the style `from cudf... import Class`.

Reply:
Yeah looks like I missed a few. I intended to just use the `from cudf... import MultiIndex` style, but only for `MultiIndex` since fixing that for all imports is a bigger task to be addressed later. In this PR I just did enough to minimize function-local imports. I'll fix the outstanding cases you pointed out.